docs/sop/gather-cluster-logs.md

```diff
@@ -0,0 +1,136 @@
+# Collect logs from Online clusters
+
+## Synopsis
+
+````
+  ssh -i online/logs_access_key opsmedic@use-tower2.ops.rhcloud.com \
+      -- -u USER -c CLUSTER [-n node1 node2...] > logs.tar.gz
+````
+
+**NOTE**: the collected logs are dumped to the standard output as a compressed
+tarball, so **you must redirect the output of ssh** to a file of your choice.
```

Comment:
Correction: you *should* redirect ... to a file.

(technically, you might also want to pipe the output into tar to unpack the files locally)